app/models/observation.rb

```diff
@@ -2660,7 +2660,7 @@ def intersecting_places
   end
 
   # Actually referring to Place::STATE_LEVEL seems to cause trouble here
-  [1, 2].each do |admin_level|
+  [10, 20].each do |admin_level|
```

Comment:
I think the problem here was either a circular reference problem, or maybe loading the Place model triggered a lot of other stuff like loading preferences that then caused problems. Could we avoid that by moving these constance into a standalone module or class?

Reply:
I didn't try any other options here because of the comment, and I wasn't sure exactly what issues it was causing. I can look into it more, but maybe after merging and deploying in this code so we can isolate this specific change